classes/opponentPhaseFakeout.lua

```diff
@@ -0,0 +1,43 @@
+-- Large cinematic with a wuss boss that dies in one shot
+
+local opponentPhaseFakeout = {}
+opponentPhaseFakeout.__index = opponentPhaseFakeout
+
+
+opponentPhaseFakeout.RADIUS = 75
+opponentPhaseFakeout.NUM_LIVES = 1
+
+
+--------------------
+-- MAIN CALLBACKS --
+--------------------
+
+function opponentPhaseFakeout.new()
+	local self = classes.opponentBase.new(opponentPhaseFakeout)
+
+	self.APPROACH_SPEED = 100
+
+	return self
+end
+
+function opponentPhaseFakeout:update(dt)
+	-- Call superclass method
+	classes.opponentBase.update(self, dt)
+
+
```

Comment:
Add `if not state.player then return end` to prevent the game from crashing on player death